samples/charger/src/main.c

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2023 Cirrus Logic, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "zephyr/sys/printk.h"
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/charger.h>
+#include <sys/errno.h>
+
+static const struct device *chgdev = DEVICE_DT_GET(DT_NODELABEL(charger));
+
+int main(void)
+{
+	union charger_propval val;
+	int ret;
+
+	if (chgdev == NULL) {
+		printk("\nError: no charger device found.\n");
```

Comment:
nit: why the leading `\n` here and below? printk normally only have a trailing one, there should not be anything on the console when this happens, if there is it should a problem of the other log/print

Reply:
ack, dropped the leading `\n`. It was not for any particular reason.